src/components/MenuItem.tsx

```diff
@@ -723,6 +723,7 @@ function MenuItem(
                                                                 )}
                                                             </View>
                                                         )}
+                                                        {titleComponent}
```

Comment:
Also please help to explain this change. Note that, this component is used in many places in our app